src/v/redpanda/application.cc

```diff
@@ -1083,6 +1086,8 @@ void application::start(::stop_signal& app_signal) {
 
 void application::start_redpanda() {
     syschecks::systemd_message("Staring storage services").get();
+    // single instance
+    storage_node.invoke_on_all(&storage::node_api::start).get0();
```

Comment:
This is the common pattern.. but it looks like a bunch of double calls to `service::start()`